airbyte-commons/src/main/java/io/airbyte/commons/exceptions/ConfigErrorException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.exceptions;
+
+public class ConfigErrorException extends RuntimeException {
+
+  private final String displayMessage;
+
+  public ConfigErrorException(final String displayMessage) {
+    super(displayMessage);
+    this.displayMessage = displayMessage;
+  }
+
+  public ConfigErrorException(final String displayMessage, final Throwable exception) {
+    super(exception);
```

Comment:
We should pass the display message in the super as well right? 
`super(displayMessage, exception);`